client/src/components/Header.css

```diff
@@ -47,3 +50,35 @@
   padding: 0px 20px;
   color: var(--color-purple);
 }
+
+.Header__navigation__link {
+  padding: 10px;
+  border: 2px var(--color-purple) solid;
+  margin: 10px 0px;
+  width: 80%;
```

Comment:
Don't know if its better to use rem here instead of px (for padding and margin)🤔

Reply:
yes, i need to fix that everywhere.. using rem for margin and em for padding...

Reply:
Ah ok, I see:D